http3/client.go

```diff
@@ -343,6 +350,9 @@ func (c *ClientConn) doRequest(req *http.Request, str *requestStream) (*http.Res
 					return nil, err
```

Comment:
This should be refactored to call `traceGot1xxResponse`.

Reply:
ok